README.md

```diff
@@ -1,22 +1,19 @@
 # A Nix expression for nightly Rust versions
 
-This is useful for working with the latest Rust nightly (or the nightly from a specific date) on [NixOS].
-Notably, it uses Nix's `builtins.fetchurl` feature to avoid the need to specify source hashes manually.
-This feature is banned in [nixpkgs], so this expression has to exist outside of it.
-
-Essentially, this expression is convenient for development but unsuitable for official packaging.
-
+This project is used for using Rust nightlies on [NixOS].
```

Comment:
> used for using

Maybe not the best wording. :)

How about: "This enables you to easily use Rust nightlies on [NixOS]. You can download the latest nightly or specify a date to get exactly the nightly you need."